src/components/projects-summary/projects-summary.html

```diff
@@ -99,4 +99,25 @@ <h2 class="secondary" ng-if="$ctrl.projects && $ctrl.projects.length > 0">Gettin
       </ul>
     </div>
   </div>
+  <div ng-if="$ctrl.recentlyViewedItems.length">
+    <h2 class="secondary">Recently Viewed</h2>
+    <div class="services-view">
+      <div class="services-view-container">
+        <div class="services-items-inner">
+          <div class="services-items-inner-shadow-covers">
+            <a href="" class="services-item" ng-repeat="item in $ctrl.recentlyViewedItems track by (item.resource | uid)"
```

Comment:
It's really odd to see them shuffle when I click on one of the recently-viewed items.

I think we should delay updating recently-viewed until after the panel opens as well because seeing them move is distracting. But I think that can be a follow-on if @sg00dwin helps with the styling.